index.html

```diff
@@ -108,7 +108,43 @@ <h2>A project for first-time contributions</h2>
       <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
       <!-- make sure there is one line of space above and below your card -->
       <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+      
+      <!-- ________ Alexannder's card START ________  -->
+      <div class="card">
+        <p class="name">Alexander Kurbatov</p>
+        <p class="contact">
+          <i class="fab fa-twitter"></i>
+          <a href="https://twitter.com/AAKurbatov" target="_blank">AAKurbatov</a>
+          <i class="fab fa-github"></i>
+          <a href="https://github.com/AlexSiberia" target="_blank">AlexSiberia</a>
+        </p>
+        <p class="about">
+          iOS Developer
+        </p>
+        <div class="resources">
+          <p>3 Useful Dev Resources</p>
+          <ul>
+            <li>
 
```

Comment:
Hello @AlexSiberia, please remove this blank line and push your commit to this PR and we should be able to merge your PR. Thank you and take care 🚀

Reply:
Hello! I did it!